docs/metadata/prophecy-build-tool/pbt-jenkins.md

```diff
@@ -0,0 +1,302 @@
+---
+title: PBT on Jenkins
+id: prophecy-build-tool-jenkins
+description: Example Usage of Prophecy Build Tool on Jenkins
+sidebar_position: 5
+tags:
+  - metadata
+  - build
+  - deploy
+  - test
+  - cli
+  - continuous integration
+  - continuous deployment
+  - cicd
+  - jenkins
+---
+
+## Context of the Jenkins CI/CD Example
+
+In this section we will explore how to set up separate "testing" and "deploying" Jenkins Jobs using declarative Pipelines. These Jobs will be triggered when items are merged into the following protected branches
+`prod`, `qa`, `develop`. Each of these three branches represents a different Databricks Workspace environment. We
+want to be able to test and deploy our Pipelines into each of these three workspaces during our release workflow.
+
+The release process on Github is defined as merging and testing to branches in the following
+order: `feature-branch` > `develop` > `qa` > `prod`
+
+![branch_protection_checks_example.png](img%2Fbranch_protection_checks_example.png)
+_When PRs are made to a protected branch, the unit tests must pass before the PR can be merged_
+
+Once the PR is merged successfully, we deploy the artifacts and Job definitions to Databricks.
+
+:::info
+
+The Git project for this example will soon be made publicly available. To keep as much
+of the CICD logic attached to the Git project itself, we will store the Jenkinsfiles
+directly in the repo and only store the Jenkins Triggers and Credentials in the Jenkins
+server.
+
+This example is currently designed for Pyspark Pipelines. To use the same recipe
+for Scala Pipelines, just make sure `JDK 11` is installed on your Jenkins nodes.
+
+:::
+
+## Preparation
+
+You should have access to:
+
+- a Git repo with an existing Prophecy Project
+- a Jenkins Server where you can create new pipelines and credentials
+
+### Jenkins plugins
+
+The following plugins were used for this example:
+
+- [Github Pull Request Builder](https://plugins.jenkins.io/ghprb/)
```

Comment:
"The Jenkins project announced unresolved security vulnerabilities affecting the current version of this plugin" fyi

Reply:
I can put in a warning, but ultimately it's an example and users don't have to use these triggers to create their pipelines

Reply:
put in a warning